MAUI/PDF-Viewer/Coordinates-Conversion.md

```diff
@@ -0,0 +1,139 @@
+
+You can obtain the PDF page coordinates using the `ConvertClientPointToPagePoint(clientPoint, pageNumber)` method by passing the client point and page number as input parameters. The following code sample explains how to convert a tapped client area position to a page point.
+
+{% tabs %}
+{% highlight c# %}
+//Subscribe the Tapped event of PDF Viewer
+PdfViewer.Tapped += PdfViewer_Tapped;
+ 
+// Handle the tapped event.
+private void PdfViewer_Tapped(object sender, GestureEventArgs e)
+{
+    //Retrieve the tapped client area position.
+    Point clientPoint = e.Position;
+    //Retrieve the page number that corresponds to the client point.
+    int pageNumber = PdfViewer.GetPageNumberFromClientPoint(clientPoint);
+    //Retrieve the page point.
+    Point pagePoint = PdfViewer.ConvertClientPointToPagePoint(clientPoint, pageNumber);
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Convert PDF page coordinates to PDF Viewer’s coordinates
+
+You can obtain the PDF Viewer’s coordinates using the `ConvertPagePointToClientPoint(pagePoint, pageNumber)` method by passing the page point and page number as input parameters. The following code sample explains how to convert a square annotation’s position in the page coordinates to a client point.
+
+{% tabs %}
+{% highlight c# %}
+// Obtain the annotation collection using `SfPdfViewer` instance.
+ReadOnlyObservableCollection<Annotation> annotations = PdfViewer.Annotations;
+
+// Obtain the first annotation in the annotation collection.
+Annotation annotation = annotations[0];
+
+// Type cast to the specific annotation type. Here the first annotation is a square annotation.
+if (annotation is SquareAnnotation squareAnnotation)
+{
+    //Get the annotation’s page number.
+    int pageNumber = squareAnnotation.PageNumber;
+    //Get the annotation bounds.
+    RectF annotationBounds = squareAnnotation.Bounds;
+    //Find the position of the annotation in page coordinates.
+    Point pagePoint = new Point(annotationBounds.X, annotationBounds.Y);
+    //Convert the page point to client point.
+    Point clientPoint = PdfViewer.ConvertPagePointToClientPoint(pagePoint, pageNumber);
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Convert PDF page coordinates to scroll coordinates
+
+You can obtain the PDF Viewer’s scroll coordinates using the `ConvertPagePointToScrollPoint(pagePoint, pageNumber)` method by passing the page point and page number as input parameters. The following code example explains how to convert a square annotation’s position in the page coordinates to a scroll point.
+
+{% tabs %}
+{% highlight c# %}
+// Obtain the annotation collection using `SfPdfViewer` instance.
+ReadOnlyObservableCollection<Annotation> annotations = PdfViewer.Annotations;
+
+// Obtain the first annotation in the annotation collection.
+Annotation annotation = annotations[0];
+
+// Type cast to the specific annotation type. Here the first annotation is a square annotation.
```

Comment:
// Type cast to the specific annotation type. Here, the first annotation is a square annotation.